app/models.py

```diff
@@ -1414,3 +1414,24 @@ def serialize(self):
             'created_at': self.created_at.strftime(DATETIME_FORMAT),
             'updated_at': self.updated_at.strftime(DATETIME_FORMAT) if self.updated_at else None
         }
+
+
+class NotificationSmsSender(db.Model):
+    __tablename__ = "notification_to_email_sender"
```

Comment:
Change this to `notification_to_email_reply_to`? Or something like that

Reply:
Ok updated